.bazelrc

```diff
@@ -11,7 +11,7 @@ build \
   --spawn_strategy=standalone \
   --verbose_failures \
   --workspace_status_command=envoy/bazel/get_workspace_status \
-  --xcode_version=10.2.1 \
+  --xcode_version=10.3 \
```

Comment:
technically if there was a 10.3.1 it would be based on the user's local environment which to use, but that's probably fine for this project

Reply:
Updated to lock to `10.3.0`